src/fragments/lib/auth/native_common/signin_with_custom_flow/common.mdx

```diff
@@ -1,49 +1,71 @@
+The Auth category can be configured to perform a [custom authentication flow](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-challenge.html) defined by you. The following guide shows how to setup a simple passwordless authentication flow.
 
 ## Prerequisites
 
-import ios0 from "/src/fragments/lib/auth/ios/getting_started/10_preReq.mdx";
+import ios0 from '/src/fragments/lib/auth/ios/getting_started/10_preReq.mdx';
 
-<Fragments fragments={{ios: ios0}} />
+<Fragments fragments={{ ios: ios0 }} />
+
+import flutter0 from '/src/fragments/lib/auth/flutter/getting_started/10_preReq.mdx';
+
+<Fragments fragments={{ flutter: flutter0 }} />
 
 ## Configure Auth Category
 
-import ios1 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/10_cli_setup.mdx";
+import ios1 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/10_cli_setup.mdx';
+
+<Fragments fragments={{ ios: ios1 }} />
 
-<Fragments fragments={{ios: ios1}} />
+import flutter1 from '/src/fragments/lib/auth/flutter/signin_with_custom_flow/10_cli_setup.mdx';
+
+<Fragments fragments={{ flutter: flutter1 }} />
 
 ## Register a user
 
 The CLI flow as mentioned above requires a username and a valid email id as parameters to register a user. Invoke the following api to initiate a sign up flow.
 
-import ios2 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/20_signup.mdx";
+import ios2 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/20_signup.mdx';
+
+<Fragments fragments={{ ios: ios2 }} />
+
+import flutter2 from '/src/fragments/lib/auth/flutter/signin_with_custom_flow/20_signup.mdx';
 
-<Fragments fragments={{ios: ios2}} />
+<Fragments fragments={{ flutter: flutter2 }} />
 
 The next step in the sign up flow is to confirm the user. A confirmation code will be sent to the email id provided during sign up. Enter the confirmation code received via email in the `confirmSignUp` call.
 
-import ios3 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/30_confirmSignup.mdx";
+import ios3 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/30_confirmSignup.mdx';
 
-<Fragments fragments={{ios: ios3}} />
+<Fragments fragments={{ ios: ios3 }} />
+
+import flutter3 from '/src/fragments/lib/auth/flutter/signin/20_confirmSignup.mdx';
+
+<Fragments fragments={{ flutter: flutter3 }} />
 
 ## Sign in a user
 
 Implement a UI to get the username from the user. After the user enters the username you can start the sign in flow by calling the following method:
 
-import ios4 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/40_signin.mdx";
+import ios4 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/40_signin.mdx';
+
+<Fragments fragments={{ ios: ios4 }} />
 
-<Fragments fragments={{ios: ios4}} />
+import flutter4 from '/src/fragments/lib/auth/flutter/signin_with_custom_flow/30_signin.mdx';
+
+<Fragments fragments={{ flutter: flutter4 }} />
 
 ## Confirm sign in with custom challenge
```

Comment:
Is this actually using a TOTP mechanism for authentication? or do I still need to use Username and password, and this is working like MFA? I am a little confused.

Reply:
This is explained elsewhere.